EbxLinkedIn/src/main/java/com/echobox/api/linkedin/types/CodeAndNameType.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.echobox.api.linkedin.types;
+
+import lombok.Getter;
+
+/**
+ * Code and name model wrapper
+ * @param <C> code type
+ * @author Joanna
+ *
+ */
+public class CodeAndNameType<C extends CodeType<?>> {
```

Comment:
Not sure why this wrapper class is required, cant we just use the enum directly?

Reply:
Just for serialization issues, if LinkedIn add a new code and name we won't be able to serialize it. Having the raw object garuntees we will have all the information. 
I'm going to have this as a raw value and then have the JSONMapper map these values to enums as I think that will be neater.
Happy to discuss.